tests/test_compile.py

```diff
@@ -1,9 +1,9 @@
-from . import c_eval
-from . import c_exec
-from . import e_eval
 from RestrictedPython import compile_restricted
 from RestrictedPython import CompileResult
 from RestrictedPython._compat import IS_PY2
+from tests import c_eval
+from tests import c_exec
+from tests import e_eval
```

Comment:
I do generally prefer absolute imports as this support readability and to understand where things come from. 

The __init__.py is something we should avoid in total, it is directly derecommended in the pytest goog practice guide: http://doc.pytest.org/en/latest/goodpractices.html#choosing-a-test-layout-import-rules But without a __init__.py there is no namespace to import from.

regarding helpers I would also prefer that, but suggest utils.py or utilities.py.